src/sql/parts/profiler/contrib/profilerActions.ts

```diff
@@ -88,7 +89,9 @@ export class ProfilerStart extends Action {
 
 export class ProfilerPause extends Action {
 	public static ID = 'profiler.pause';
-	public static LABEL = nls.localize('pause', "Pause");
+	public static LABEL = nls.localize('pause', "Capture");
```

Comment:
Is the label always `Capture` even when already paused? That seems fine to unpause but probably isn't great to pause. Maybe `Pause` or `Pause Capture` to pause and `Capture` to unpause?

Reply:
I compromised on that by changing the icon, so it has a play or stop icon with the phrase capture. I can definitely change that though, would you say "pause capture" and "capture" or "pause capture" and "resume capture"?

Reply:
Probably "pause capture" and "resume capture" as long as those aren't too long to look ok in the toolbar